### PR TITLE
shah-10

### PR DESCRIPTION
**Pull request by nevgh**
added links to social media icons

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -21,10 +21,16 @@ const Footer = () => {
       </div>
       <div>
         <h5>FOLLOW US</h5>
-        <div>
-          <FaInstagramSquare />
-          <FaTwitterSquare />
-          <FaFacebookSquare />
+        <div className="social-media-icons">
+          <a href="https://www.instagram.com">
```
Comment (sargonsaadi) [inline src/components/Footer.js]: Nice. This works, but let's open the link in a new browser tab. 
Currently it's openning in the same tab, and we're losing out website.

Comment (nevgh) [reply src/components/Footer.js]: okay got that fixed.

File: src/components/Footer.js [hunk 2]
```diff
@@ -22,13 +22,13 @@ const Footer = () => {
       <div>
         <h5>FOLLOW US</h5>
         <div className="social-media-icons">
-          <a href="https://www.instagram.com">
+          <a href="https://www.instagram.com" target="_blank">
             <FaInstagramSquare />
           </a>
-          <a href="https://twitter.com/">
+          <a href="https://twitter.com/" target="_blank">
```
Comment (sargonsaadi) [inline src/components/Footer.js:28]: Perfect.

**Review by sargonsaadi — APPROVED** [review]

**Comment by nevgh** [comment]
social media links open in new tab and logo redirects to home page once it's clicked

**Pull request merged**